pkgs/tools/system/tuptime/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "tuptime";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "rfrail3";
+    repo = "tuptime";
+    rev = version;
+    sha256 = "0p5v1jp6bl0hjv04q3gh11q6dx9z0x61h6svcbvwp5ni0h1bkz1a";
+  };
+
+  buildInputs = [ python3 ];
+
+  installPhase = ''
+    mkdir -p "$out/bin"
```

Comment:
you can remove these and below doublequotes - nix store paths don't contain spaces.

Reply:
i was wondering about that, seems like most packages quote all paths